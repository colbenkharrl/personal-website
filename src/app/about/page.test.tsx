import { render, screen } from '@testing-library/react';
import Page from './page';

it('should match page snapshot', () => {
  render(<Page />);

  expect(screen.getByTestId('Container')).toMatchSnapshot('page snapshot');
});
