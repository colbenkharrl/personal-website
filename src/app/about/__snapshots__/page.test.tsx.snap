// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match page snapshot: page snapshot 1`] = `
<div
  class="sm:px-8 mt-16 sm:mt-32"
  data-testid="Container"
>
  <div
    class="mx-auto w-full max-w-7xl lg:px-8"
  >
    <div
      class="relative px-4 sm:px-8 lg:px-12"
    >
      <div
        class="mx-auto max-w-2xl lg:max-w-5xl"
      >
        <div
          class="grid grid-cols-1 gap-y-16 lg:grid-cols-2 lg:grid-rows-[auto_1fr] lg:gap-y-12"
        >
          <div
            class="lg:pl-20"
          >
            <div
              class="max-w-xs px-2.5 lg:max-w-none"
            >
              <img
                alt=""
                class="aspect-square rounded-2xl bg-zinc-100 object-cover dark:bg-zinc-800"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                sizes="(min-width: 1024px) 32rem, 20rem"
                src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=16&q=75 16w, /_next/image?url=%2Fimg.jpg&w=32&q=75 32w, /_next/image?url=%2Fimg.jpg&w=48&q=75 48w, /_next/image?url=%2Fimg.jpg&w=64&q=75 64w, /_next/image?url=%2Fimg.jpg&w=96&q=75 96w, /_next/image?url=%2Fimg.jpg&w=128&q=75 128w, /_next/image?url=%2Fimg.jpg&w=256&q=75 256w, /_next/image?url=%2Fimg.jpg&w=384&q=75 384w, /_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
                style="color: transparent;"
                width="40"
              />
            </div>
          </div>
          <div
            class="lg:order-first lg:row-span-2"
          >
            <h1
              class="text-4xl font-bold tracking-tight text-zinc-800 dark:text-zinc-100 sm:text-4xl"
            >
              I’m Matt Kharrl: 
              <br />
               
              <span
                class="text-yellow-500 dark:text-yellow-400"
              >
                engineer
              </span>
              , 
              <span
                class="text-sky-500 dark:text-sky-400"
              >
                athlete
              </span>
              , and
               
              <span
                class="text-emerald-500 dark:text-emerald-400"
              >
                dad
              </span>
              .
            </h1>
            <div
              class="mt-6 space-y-7 text-base text-zinc-600 dark:text-zinc-400"
            >
              <p>
                I like to approach everything I do in life with genuine curiosity and a sense of adventure. If I had to boil it down, however, my passions would fall into these three categories.
              </p>
              <h2
                class="text-2xl font-bold tracking-tight text-zinc-800 dark:text-zinc-100 sm:text-2xl"
              >
                I build
                 
                <span
                  class="text-yellow-500 dark:text-yellow-400"
                >
                  software
                </span>
                .
              </h2>
              <p>
                I knew I wanted to spend my life immersed in technology by the age of 15. My journey since then has taken me from building websites for friends to studying computer science at Arizona State University to building web applications for some of the largest technology companies on the planet to building and launching a startup company in Silicon Valley from the ground up. Every experience has taught me valuable lessons about building efficient and practical software at every scale. While I have worked in many roles across the stack and development lifecycle, I find myself the happiest when creating web experiences as close to end users as possible.
              </p>
              <a
                href="/portfolio"
              >
                <span
                  class="mt-4 inline-flex items-center rounded-md bg-yellow-50 px-2 py-1 text-xs font-medium text-yellow-600 ring-1 ring-inset ring-yellow-500/10 hover:bg-yellow-100 dark:bg-yellow-400/10 dark:text-yellow-400 dark:ring-yellow-400/30 dark:hover:bg-yellow-400/20"
                >
                  <svg
                    aria-hidden="true"
                    class="mr-1.5 h-5 w-5"
                    fill="none"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.5"
                    viewBox="0 0 24 24"
                  >
                    <path
                      class="stroke-yellow-600 dark:stroke-yellow-400"
                      d="M2.75 9.75a3 3 0 0 1 3-3h12.5a3 3 0 0 1 3 3v8.5a3 3 0 0 1-3 3H5.75a3 3 0 0 1-3-3v-8.5Z"
                    />
                    <path
                      class="stroke-yellow-600 dark:stroke-yellow-400"
                      d="M3 14.25h6.249c.484 0 .952-.002 1.316.319l.777.682a.996.996 0 0 0 1.316 0l.777-.682c.364-.32.832-.319 1.316-.319H21M8.75 6.5V4.75a2 2 0 0 1 2-2h2.5a2 2 0 0 1 2 2V6.5"
                    />
                  </svg>
                  <div
                    class="flex items-center justify-center"
                  >
                    <p
                      class="whitespace-nowrap"
                    >
                      View My Portfolio
                       
                      <span
                        aria-hidden="true"
                        class="ml-1"
                      >
                        →
                      </span>
                    </p>
                  </div>
                </span>
              </a>
              <h2
                class="text-2xl font-bold tracking-tight text-zinc-800 dark:text-zinc-100 sm:text-2xl"
              >
                I race
                 
                <span
                  class="text-sky-500 dark:text-sky-400"
                >
                  bikes
                </span>
                .
              </h2>
              <p>
                Bikes and I go way back. It's been a long and lovely relationship, from cruising around suburban neighborhoods as a kid to learning tricks on BMX bikes as a teen to skidding to class on fixed-gear bikes in college to exploring nature with a camera strapped to my back in California to lately venturing into the world of racing and growing as an athlete in the Arizona cycling community. Bikes strengthen my bond with nature, my connection with my community, and my understanding of myself. I currently race with Blitz Racing and spend much of my free time training and riding with teammates and friends.
              </p>
              <h2
                class="text-md sm:text-md font-bold tracking-tight text-zinc-800 dark:text-zinc-100"
              >
                Recent accomplishments:
              </h2>
              <ol>
                <li>
                  <a
                    class="flex items-center stroke-sky-500 hover:text-sky-500 dark:stroke-sky-400 dark:hover:text-sky-400"
                    href="https://legacy.usacycling.org/results/index.php?permit=2023-9995"
                  >
                    <svg
                      class="mr-2 h-6 w-6"
                      fill="none"
                      stroke-width="1.5"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M16.5 18.75h-9m9 0a3 3 0 013 3h-15a3 3 0 013-3m9 0v-3.375c0-.621-.503-1.125-1.125-1.125h-.871M7.5 18.75v-3.375c0-.621.504-1.125 1.125-1.125h.872m5.007 0H9.497m5.007 0a7.454 7.454 0 01-.982-3.172M9.497 14.25a7.454 7.454 0 00.981-3.172M5.25 4.236c-.982.143-1.954.317-2.916.52A6.003 6.003 0 007.73 9.728M5.25 4.236V4.5c0 2.108.966 3.99 2.48 5.228M5.25 4.236V2.721C7.456 2.41 9.71 2.25 12 2.25c2.291 0 4.545.16 6.75.47v1.516M7.73 9.728a6.726 6.726 0 002.748 1.35m8.272-6.842V4.5c0 2.108-.966 3.99-2.48 5.228m2.48-5.492a46.32 46.32 0 012.916.52 6.003 6.003 0 01-5.395 4.972m0 0a6.726 6.726 0 01-2.749 1.35m0 0a6.772 6.772 0 01-3.044 0"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    1st - AZ TTT Championship (2x)
                  </a>
                </li>
                <li>
                  <a
                    class="flex items-center stroke-sky-500 hover:text-sky-500 dark:stroke-sky-400 dark:hover:text-sky-400"
                    href="https://legacy.usacycling.org/results/index.php?permit=2023-9543"
                  >
                    <svg
                      class="mr-2 h-6 w-6"
                      fill="none"
                      stroke-width="1.5"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M16.5 18.75h-9m9 0a3 3 0 013 3h-15a3 3 0 013-3m9 0v-3.375c0-.621-.503-1.125-1.125-1.125h-.871M7.5 18.75v-3.375c0-.621.504-1.125 1.125-1.125h.872m5.007 0H9.497m5.007 0a7.454 7.454 0 01-.982-3.172M9.497 14.25a7.454 7.454 0 00.981-3.172M5.25 4.236c-.982.143-1.954.317-2.916.52A6.003 6.003 0 007.73 9.728M5.25 4.236V4.5c0 2.108.966 3.99 2.48 5.228M5.25 4.236V2.721C7.456 2.41 9.71 2.25 12 2.25c2.291 0 4.545.16 6.75.47v1.516M7.73 9.728a6.726 6.726 0 002.748 1.35m8.272-6.842V4.5c0 2.108-.966 3.99-2.48 5.228m2.48-5.492a46.32 46.32 0 012.916.52 6.003 6.003 0 01-5.395 4.972m0 0a6.726 6.726 0 01-2.749 1.35m0 0a6.772 6.772 0 01-3.044 0"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    1st - David Gomez Road Race
                  </a>
                </li>
                <li>
                  <a
                    class="flex items-center stroke-sky-500 hover:text-sky-500 dark:stroke-sky-400 dark:hover:text-sky-400"
                    href="https://legacy.usacycling.org/results/index.php?permit=2023-9336"
                  >
                    <svg
                      class="mr-2 h-6 w-6"
                      fill="none"
                      stroke-width="1.5"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M16.5 18.75h-9m9 0a3 3 0 013 3h-15a3 3 0 013-3m9 0v-3.375c0-.621-.503-1.125-1.125-1.125h-.871M7.5 18.75v-3.375c0-.621.504-1.125 1.125-1.125h.872m5.007 0H9.497m5.007 0a7.454 7.454 0 01-.982-3.172M9.497 14.25a7.454 7.454 0 00.981-3.172M5.25 4.236c-.982.143-1.954.317-2.916.52A6.003 6.003 0 007.73 9.728M5.25 4.236V4.5c0 2.108.966 3.99 2.48 5.228M5.25 4.236V2.721C7.456 2.41 9.71 2.25 12 2.25c2.291 0 4.545.16 6.75.47v1.516M7.73 9.728a6.726 6.726 0 002.748 1.35m8.272-6.842V4.5c0 2.108-.966 3.99-2.48 5.228m2.48-5.492a46.32 46.32 0 012.916.52 6.003 6.003 0 01-5.395 4.972m0 0a6.726 6.726 0 01-2.749 1.35m0 0a6.772 6.772 0 01-3.044 0"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    1st - Flapjack Flats ITT
                  </a>
                </li>
                <li>
                  <a
                    class="flex items-center stroke-sky-500 hover:text-sky-500 dark:stroke-sky-400 dark:hover:text-sky-400"
                    href="https://legacy.usacycling.org/results/index.php?permit=2023-10667"
                  >
                    <svg
                      class="mr-2 h-6 w-6"
                      fill="none"
                      stroke-width="1.5"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M16.5 18.75h-9m9 0a3 3 0 013 3h-15a3 3 0 013-3m9 0v-3.375c0-.621-.503-1.125-1.125-1.125h-.871M7.5 18.75v-3.375c0-.621.504-1.125 1.125-1.125h.872m5.007 0H9.497m5.007 0a7.454 7.454 0 01-.982-3.172M9.497 14.25a7.454 7.454 0 00.981-3.172M5.25 4.236c-.982.143-1.954.317-2.916.52A6.003 6.003 0 007.73 9.728M5.25 4.236V4.5c0 2.108.966 3.99 2.48 5.228M5.25 4.236V2.721C7.456 2.41 9.71 2.25 12 2.25c2.291 0 4.545.16 6.75.47v1.516M7.73 9.728a6.726 6.726 0 002.748 1.35m8.272-6.842V4.5c0 2.108-.966 3.99-2.48 5.228m2.48-5.492a46.32 46.32 0 012.916.52 6.003 6.003 0 01-5.395 4.972m0 0a6.726 6.726 0 01-2.749 1.35m0 0a6.772 6.772 0 01-3.044 0"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    2nd - AZ ITT Championship
                  </a>
                </li>
                <li>
                  <a
                    class="flex items-center stroke-sky-500 hover:text-sky-500 dark:stroke-sky-400 dark:hover:text-sky-400"
                    href="https://results.raceroster.com/v2/en-US/results/vrav9hjwkkbdtfem/results"
                  >
                    <svg
                      class="mr-2 h-6 w-6"
                      fill="none"
                      stroke-width="1.5"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M16.5 18.75h-9m9 0a3 3 0 013 3h-15a3 3 0 013-3m9 0v-3.375c0-.621-.503-1.125-1.125-1.125h-.871M7.5 18.75v-3.375c0-.621.504-1.125 1.125-1.125h.872m5.007 0H9.497m5.007 0a7.454 7.454 0 01-.982-3.172M9.497 14.25a7.454 7.454 0 00.981-3.172M5.25 4.236c-.982.143-1.954.317-2.916.52A6.003 6.003 0 007.73 9.728M5.25 4.236V4.5c0 2.108.966 3.99 2.48 5.228M5.25 4.236V2.721C7.456 2.41 9.71 2.25 12 2.25c2.291 0 4.545.16 6.75.47v1.516M7.73 9.728a6.726 6.726 0 002.748 1.35m8.272-6.842V4.5c0 2.108-.966 3.99-2.48 5.228m2.48-5.492a46.32 46.32 0 012.916.52 6.003 6.003 0 01-5.395 4.972m0 0a6.726 6.726 0 01-2.749 1.35m0 0a6.772 6.772 0 01-3.044 0"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    2nd - AZ Road Race Championship
                  </a>
                </li>
                <li>
                  <a
                    class="flex items-center stroke-sky-500 hover:text-sky-500 dark:stroke-sky-400 dark:hover:text-sky-400"
                    href="https://www.strava.com/athletes/41283083"
                  >
                    <svg
                      class="mr-2 h-6 w-6"
                      fill="none"
                      stroke-width="1.5"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M6.115 5.19l.319 1.913A6 6 0 008.11 10.36L9.75 12l-.387.775c-.217.433-.132.956.21 1.298l1.348 1.348c.21.21.329.497.329.795v1.089c0 .426.24.815.622 1.006l.153.076c.433.217.956.132 1.298-.21l.723-.723a8.7 8.7 0 002.288-4.042 1.087 1.087 0 00-.358-1.099l-1.33-1.108c-.251-.21-.582-.299-.905-.245l-1.17.195a1.125 1.125 0 01-.98-.314l-.295-.295a1.125 1.125 0 010-1.591l.13-.132a1.125 1.125 0 011.3-.21l.603.302a.809.809 0 001.086-1.086L14.25 7.5l1.256-.837a4.5 4.5 0 001.528-1.732l.146-.292M6.115 5.19A9 9 0 1017.18 4.64M6.115 5.19A8.965 8.965 0 0112 3c1.929 0 3.716.607 5.18 1.64"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    Biked the Circumference of Earth
                  </a>
                </li>
              </ol>
              <h2
                class="text-2xl font-bold tracking-tight text-zinc-800 dark:text-zinc-100 sm:text-2xl"
              >
                I love my
                 
                <span
                  class="text-emerald-500 dark:text-emerald-400"
                >
                  family
                </span>
                .
              </h2>
              <p>
                I moved back to Arizona in 2020 to follow the love of my life. We have since been married, and in 2023, we brought our son into the world. My duty and love of being a husband and father motivates everything I do, and I honestly don't know where I would be without them. My ultimate goals are to see the world with my family and to nourish a life where everyone feels like they have the environment and encouragement to follow their dreams. That said, we don't take ourselves too seriously and spend most of our time laughing. I aspire to master the art of dad jokes at the expense of my wife and son's sanity.
              </p>
              <a
                class="flex items-center fill-emerald-500 hover:text-emerald-500 dark:fill-emerald-400 dark:hover:text-emerald-400"
                href="https://www.instagram.com/matt.kharrl/"
              >
                <svg
                  aria-hidden="true"
                  class="mr-2 h-6 w-6"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 3c-2.444 0-2.75.01-3.71.054-.959.044-1.613.196-2.185.418A4.412 4.412 0 0 0 4.51 4.511c-.5.5-.809 1.002-1.039 1.594-.222.572-.374 1.226-.418 2.184C3.01 9.25 3 9.556 3 12s.01 2.75.054 3.71c.044.959.196 1.613.418 2.185.23.592.538 1.094 1.039 1.595.5.5 1.002.808 1.594 1.038.572.222 1.226.374 2.184.418C9.25 20.99 9.556 21 12 21s2.75-.01 3.71-.054c.959-.044 1.613-.196 2.185-.419a4.412 4.412 0 0 0 1.595-1.038c.5-.5.808-1.002 1.038-1.594.222-.572.374-1.226.418-2.184.044-.96.054-1.267.054-3.711s-.01-2.75-.054-3.71c-.044-.959-.196-1.613-.419-2.185A4.412 4.412 0 0 0 19.49 4.51c-.5-.5-1.002-.809-1.594-1.039-.572-.222-1.226-.374-2.184-.418C14.75 3.01 14.444 3 12 3Zm0 1.622c2.403 0 2.688.009 3.637.052.877.04 1.354.187 1.67.31.421.163.72.358 1.036.673.315.315.51.615.673 1.035.123.317.27.794.31 1.671.043.95.052 1.234.052 3.637s-.009 2.688-.052 3.637c-.04.877-.187 1.354-.31 1.67-.163.421-.358.72-.673 1.036a2.79 2.79 0 0 1-1.035.673c-.317.123-.794.27-1.671.31-.95.043-1.234.052-3.637.052s-2.688-.009-3.637-.052c-.877-.04-1.354-.187-1.67-.31a2.789 2.789 0 0 1-1.036-.673 2.79 2.79 0 0 1-.673-1.035c-.123-.317-.27-.794-.31-1.671-.043-.95-.052-1.234-.052-3.637s.009-2.688.052-3.637c.04-.877.187-1.354.31-1.67.163-.421.358-.72.673-1.036.315-.315.615-.51 1.035-.673.317-.123.794-.27 1.671-.31.95-.043 1.234-.052 3.637-.052Z"
                  />
                  <path
                    d="M12 15a3 3 0 1 1 0-6 3 3 0 0 1 0 6Zm0-7.622a4.622 4.622 0 1 0 0 9.244 4.622 4.622 0 0 0 0-9.244Zm5.884-.182a1.08 1.08 0 1 1-2.16 0 1.08 1.08 0 0 1 2.16 0Z"
                  />
                </svg>
                Keep up with us on Instagram.
              </a>
            </div>
          </div>
          <div
            class="lg:pl-20"
          >
            <ul
              role="list"
            >
              <li
                class="mt-4 flex"
              >
                <a
                  class="group flex text-sm font-medium text-zinc-800 transition hover:text-yellow-500 dark:text-zinc-200 dark:hover:text-yellow-500"
                  href="https://linkedin.com/in/mattkharrl/"
                >
                  <svg
                    aria-hidden="true"
                    class="h-6 w-6 flex-none fill-zinc-500 transition group-hover:fill-yellow-500"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M18.335 18.339H15.67v-4.177c0-.996-.02-2.278-1.39-2.278-1.389 0-1.601 1.084-1.601 2.205v4.25h-2.666V9.75h2.56v1.17h.035c.358-.674 1.228-1.387 2.528-1.387 2.7 0 3.2 1.778 3.2 4.091v4.715zM7.003 8.575a1.546 1.546 0 01-1.548-1.549 1.548 1.548 0 111.547 1.549zm1.336 9.764H5.666V9.75H8.34v8.589zM19.67 3H4.329C3.593 3 3 3.58 3 4.297v15.406C3 20.42 3.594 21 4.328 21h15.338C20.4 21 21 20.42 21 19.703V4.297C21 3.58 20.4 3 19.666 3h.003z"
                    />
                  </svg>
                  <span
                    class="ml-4"
                  >
                    Follow on LinkedIn
                  </span>
                </a>
              </li>
              <li
                class="mt-4 flex"
              >
                <a
                  class="group flex text-sm font-medium text-zinc-800 transition hover:text-yellow-500 dark:text-zinc-200 dark:hover:text-yellow-500"
                  href="https://github.com/colbenkharrl"
                >
                  <svg
                    aria-hidden="true"
                    class="h-6 w-6 flex-none fill-zinc-500 transition group-hover:fill-yellow-500"
                    viewBox="0 0 24 24"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M12 2C6.475 2 2 6.588 2 12.253c0 4.537 2.862 8.369 6.838 9.727.5.09.687-.218.687-.487 0-.243-.013-1.05-.013-1.91C7 20.059 6.35 18.957 6.15 18.38c-.113-.295-.6-1.205-1.025-1.448-.35-.192-.85-.667-.013-.68.788-.012 1.35.744 1.538 1.051.9 1.551 2.338 1.116 2.912.846.088-.666.35-1.115.638-1.371-2.225-.256-4.55-1.14-4.55-5.062 0-1.115.387-2.038 1.025-2.756-.1-.256-.45-1.307.1-2.717 0 0 .837-.269 2.75 1.051.8-.23 1.65-.346 2.5-.346.85 0 1.7.115 2.5.346 1.912-1.333 2.75-1.05 2.75-1.05.55 1.409.2 2.46.1 2.716.637.718 1.025 1.628 1.025 2.756 0 3.934-2.337 4.806-4.562 5.062.362.32.675.936.675 1.897 0 1.371-.013 2.473-.013 2.82 0 .268.188.589.688.486a10.039 10.039 0 0 0 4.932-3.74A10.447 10.447 0 0 0 22 12.253C22 6.588 17.525 2 12 2Z"
                      fill-rule="evenodd"
                    />
                  </svg>
                  <span
                    class="ml-4"
                  >
                    Follow on GitHub
                  </span>
                </a>
              </li>
              <li
                class="mt-4 flex"
              >
                <a
                  class="group flex text-sm font-medium text-zinc-800 transition hover:text-yellow-500 dark:text-zinc-200 dark:hover:text-yellow-500"
                  href="https://www.instagram.com/matt.kharrl/"
                >
                  <svg
                    aria-hidden="true"
                    class="h-6 w-6 flex-none fill-zinc-500 transition group-hover:fill-yellow-500"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 3c-2.444 0-2.75.01-3.71.054-.959.044-1.613.196-2.185.418A4.412 4.412 0 0 0 4.51 4.511c-.5.5-.809 1.002-1.039 1.594-.222.572-.374 1.226-.418 2.184C3.01 9.25 3 9.556 3 12s.01 2.75.054 3.71c.044.959.196 1.613.418 2.185.23.592.538 1.094 1.039 1.595.5.5 1.002.808 1.594 1.038.572.222 1.226.374 2.184.418C9.25 20.99 9.556 21 12 21s2.75-.01 3.71-.054c.959-.044 1.613-.196 2.185-.419a4.412 4.412 0 0 0 1.595-1.038c.5-.5.808-1.002 1.038-1.594.222-.572.374-1.226.418-2.184.044-.96.054-1.267.054-3.711s-.01-2.75-.054-3.71c-.044-.959-.196-1.613-.419-2.185A4.412 4.412 0 0 0 19.49 4.51c-.5-.5-1.002-.809-1.594-1.039-.572-.222-1.226-.374-2.184-.418C14.75 3.01 14.444 3 12 3Zm0 1.622c2.403 0 2.688.009 3.637.052.877.04 1.354.187 1.67.31.421.163.72.358 1.036.673.315.315.51.615.673 1.035.123.317.27.794.31 1.671.043.95.052 1.234.052 3.637s-.009 2.688-.052 3.637c-.04.877-.187 1.354-.31 1.67-.163.421-.358.72-.673 1.036a2.79 2.79 0 0 1-1.035.673c-.317.123-.794.27-1.671.31-.95.043-1.234.052-3.637.052s-2.688-.009-3.637-.052c-.877-.04-1.354-.187-1.67-.31a2.789 2.789 0 0 1-1.036-.673 2.79 2.79 0 0 1-.673-1.035c-.123-.317-.27-.794-.31-1.671-.043-.95-.052-1.234-.052-3.637s.009-2.688.052-3.637c.04-.877.187-1.354.31-1.67.163-.421.358-.72.673-1.036.315-.315.615-.51 1.035-.673.317-.123.794-.27 1.671-.31.95-.043 1.234-.052 3.637-.052Z"
                    />
                    <path
                      d="M12 15a3 3 0 1 1 0-6 3 3 0 0 1 0 6Zm0-7.622a4.622 4.622 0 1 0 0 9.244 4.622 4.622 0 0 0 0-9.244Zm5.884-.182a1.08 1.08 0 1 1-2.16 0 1.08 1.08 0 0 1 2.16 0Z"
                    />
                  </svg>
                  <span
                    class="ml-4"
                  >
                    Follow on Instagram
                  </span>
                </a>
              </li>
              <li
                class="mt-4 flex"
              >
                <a
                  class="group flex text-sm font-medium text-zinc-800 transition hover:text-yellow-500 dark:text-zinc-200 dark:hover:text-yellow-500"
                  href="https://www.strava.com/athletes/41283083"
                >
                  <svg
                    aria-hidden="true"
                    class="h-6 w-6 flex-none fill-zinc-500 transition group-hover:fill-yellow-500"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.387 17.944l-2.089-4.116h-3.065L15.387 24l5.15-10.172h-3.066m-7.008-5.599l2.836 5.598h4.172L10.463 0l-7 13.828h4.169"
                    />
                  </svg>
                  <span
                    class="ml-4"
                  >
                    Follow on Strava
                  </span>
                </a>
              </li>
              <li
                class="mt-8 border-t border-zinc-100 pt-8 dark:border-zinc-700/40 flex"
              >
                <a
                  class="group flex text-sm font-medium text-zinc-800 transition hover:text-yellow-500 dark:text-zinc-200 dark:hover:text-yellow-500"
                  href="mailto:matt@kharrl.com"
                >
                  <svg
                    aria-hidden="true"
                    class="h-6 w-6 flex-none fill-zinc-500 transition group-hover:fill-yellow-500"
                    fill="none"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.5"
                    viewBox="0 0 24 24"
                  >
                    <path
                      class="fill-zinc-100 stroke-zinc-400 dark:fill-zinc-100/10 dark:stroke-zinc-500"
                      d="M2.75 7.75a3 3 0 0 1 3-3h12.5a3 3 0 0 1 3 3v8.5a3 3 0 0 1-3 3H5.75a3 3 0 0 1-3-3v-8.5Z"
                    />
                    <path
                      class="stroke-zinc-400 dark:stroke-zinc-500"
                      d="m4 6 6.024 5.479a2.915 2.915 0 0 0 3.952 0L20 6"
                    />
                  </svg>
                  <span
                    class="ml-4"
                  >
                    matt@kharrl.com
                  </span>
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
